R/runexample.R

```diff
@@ -60,12 +60,42 @@ launchApp <- function(appname) {
   shiny::runApp(appDir, display.mode = "normal")
 }
 
+#' aboutApp : get more information about an App !
+#'
+#' @param appname string of which app you want to get info about
+#' This function opens the description of any given app
+#'
+#' @export
+aboutApp <- function(appname){
+
+  # locate all the shiny app examples that exist
+  validExamples <- list.files(system.file("shinys", package = "ScPoEconometrics"))
+
+  validExamplesMsg <-
+    paste0(
+      "Valid apps are: '",
+      paste(validExamples, collapse = "', '"),
+      "'")
+
+  # if an invalid example is given, throw an error
+  if (missing(appname) || !nzchar(appname) ||
+      !appname %in% validExamples) {
+    stop(
+      'Please run `launchApp()` with a valid app as an argument.\n',
+      validExamplesMsg,
+      call. = FALSE)
+  }
+
+  # render and launch the about document
+  appDir = system.file("shinys", appname, package = "ScPoEconometrics")
+  rmarkdown::render(paste0(appDir, "/about.Rmd"))
```

Comment:
i think if you do `rmarkdown::run` it pops up the browser automatically? i would like to avoid leaving any files lingering around (like, what happens to that `about.html` after you are done?)

Reply:
fixed this